### DISCUSSION TITLE
In Quick app (WordPress) is there a way to preload plugins and theme?

### DISCUSSION THREAD

**Discussion by fernandodilland** [leokhoa/laragon#379] [General]
The "Quick app" tool to install WordPress locally is wonderful, is there a way I can set up the pre-installation of specific plugins and a specific theme and activate them?

**Comment by Pen-y-Fan** [comment]
Possibly by using the WordPress CLI:  <https://developer.wordpress.org/cli/commands/plugin/install/>

**Comment by solagirl** [comment]
I created a windows batch file to automate wordpress installation. 

I used composer to install wp cli, made sure composer was run by php 7.x.

Then I installed git bash and use its terminal to run the bat file.

The script can automatically install Wordpress, delete default plugins and themes then install theme and plugins that I chosed. 

The only problem is that I have to manually restart laragon to make the new domain work. The reload command is not working as expected.   It would be nice to have a command like laragon stop and laragon start.

**Reply by Pen-y-Fan** [reply]
There is a command line, which will restart Apache or Nginx: [Laragon > Docs > Command Line](https://laragon.org/docs/cli.html)

**Reply by Pen-y-Fan** [reply]
There is also a laragon reload:

```sh
%laragon_root%\laragon reload
```

**Reply by solagirl** [reply]
Hi, I tried the example in the document, when I accessed _http://awesome.test_ it showed the content of the localhost home page. Just like the site conf file did not take effect. It works if I manually reload nginx via laragon menu.

I'm using the portable version but I also tried the full version on another windows 10 computer, same result. Anyway I am satisfied with the current results. Especially when I found that the laragon terminal can work with wp cli without any problem. I no longer need git bash command line.

**Reply by Babayaga0091** [reply]
> I created a windows batch file to automate wordpress installation.
> 
> I used composer to install wp cli, made sure composer was run by php 7.x.
> 
> Then I installed git bash and use its terminal to run the bat file.
> 
> The script can automatically install Wordpress, delete default plugins and themes then install theme and plugins that I chosed.
> 
> The only problem is that I have to manually restart laragon to make the new domain work. The reload command is not working as expected. It would be nice to have a command like laragon stop and laragon start.

Can you share your .bat file for me to refer? Thank you very much

**Reply by solagirl** [reply]
Of course, please check https://gist.github.com/solagirl/206c2ef6f9db9d18187045fef60ac821
There is an error in my last comment. Laragon terminal does not support sh , so some of the wp cli commands cannot run in laragon, like the one with xargs in use.

**Comment by andyg2** [comment]
There's also this script that I created for this

https://github.com/andyg2/wp-quick

There's also a [demo here](https://dgte.pro/dev/wp-quick/) but you should not trust my demo, I could load anything in there so recommend setting it up yourself.
